admin/users/views.py

```diff
@@ -148,6 +149,16 @@ def form_valid(self, form):
 
             return redirect(reverse('users:user', kwargs={'guid': guid}))
 
+        if orcid:
+            user = OSFUser.objects.filter(social__orcid=orcid).first()
```

Comment:
We don't want to use the social field here, because that's a user-filled field and not required for folks who use the system. We have connected identities that we use for this instead, because that means that if a user uses Single Sign On with Orcid, their info will be stored there.